metron-stellar/stellar-common/README.md

```diff
@@ -257,7 +258,7 @@ In the core language functions, we support basic functional programming primitiv
   * Description: Adds an element to the bloom filter passed in
   * Input:
     * bloom - The bloom filter
-    * value* - The values to add
+    * value\*? - The values to add
```

Comment:
Why does this have an \*?

Reply:
It's a repetition symbol, so it implies you can have any number of values.

Reply:
Okay, I get your intent now.  It was being suppressed before because it wasn't escaped, I will try to make it more obvious.